universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -357,6 +386,41 @@ private ContainerTag questionBankPanel(
     return qb.getContainer();
   }
 
+  private Modal blockDescriptionModal(Tag csrfTag, BlockForm blockForm, String blockUpdateAction) {
+    ContainerTag blockDescriptionForm =
+        form(csrfTag).withMethod(POST).withAction(blockUpdateAction);
+    blockDescriptionForm
+        .withId("block-edit-form")
+        .with(
+            div(
+                    FieldWithLabel.input()
+                        .setId("block-name-input")
+                        .setFieldName("name")
+                        .setLabelText("Block name")
+                        .setValue(blockForm.getName())
+                        .getContainer(),
+                    FieldWithLabel.textArea()
+                        .setId("block-description-textarea")
+                        .setFieldName("description")
+                        .setLabelText("Block description")
+                        .setValue(blockForm.getDescription())
+                        .getContainer())
+                .withClasses(Styles.MX_4),
+            submitButton("Save")
+                .withId("update-block-button")
+                .withClasses(
+                    Styles.MX_4,
+                    Styles.MY_1,
+                    Styles.INLINE,
+                    Styles.OPACITY_100,
+                    StyleUtils.disabled(Styles.OPACITY_50))
+                .attr("disabled", ""));
+    return Modal.builder("block-description-modal", blockDescriptionForm)
+        .setModalTitle("Block Name and Description")
+        .setButtonText("Edit Name and Description")
```

Comment:
Can you remove strings from the code flow? I know that we currently don't plan on localizing the admin view, but if we do, or if we want to go back and change a string value, it's a lot more manageable if these are static strings or at least declared at the top of the method body.

Reply:
fixed